web/static/js/components/surveys/SurveyWizardPanelSurveyStep.jsx

```diff
@@ -26,8 +30,8 @@ class SurveyWizardPanelSurveyStep extends Component {
             <p>
               <div className='switch'>
                 <label>
-                  <input type='checkbox' disabled={readOnly} checked={!!survey.panelSurveyId} onChange={() => {
-                    // TODO: change the new property new `generates_panel_survey` property here
+                  <input type='checkbox' disabled={readOnly} checked={generatesPanelSurvey} onChange={() => {
+                    dispatch(generatesPanelSurveySwitched(!generatesPanelSurvey))
```

Comment:
why is the `generatePanelSurvey` negated?

Reply:
Because `generatesPanelSurveySwitched` receives the new value. But please talk about this in person, because it's too basic and  I'm afraid that chatting we could make it bigger than it deserves 🙃